server/JsDbg.Gdb/GdbDebugger.cs

```diff
@@ -120,11 +120,38 @@ class GdbDebugger : IDebugger {
         }
         
         public async Task<IEnumerable<SConstantResult>> LookupConstants(string module, string type, ulong constantValue) {
-            // Not currently aware of a way to find all constants with a given value from GDB
-            // TODO:
-            //   -var-create - * ((type)constantValue)
-            //   may return a useful result; ^done,name="var1",numchild="0",value="ENUM_1",type="enum1",has_more="0"
-            return new List<SConstantResult>();
+            string pythonResult = await this.QueryDebuggerPython(String.Format("LookupConstants(\"{0}\", \"{1}\", {2})", module, type, constantValue));
+
+            List<SConstantResult> result = new List<SConstantResult>();
+
+            // TODO: Refactor this list parsing to share code with GetAllFields
+            // [{...}, {...}]

```

Comment:
I put it here, for consistency with where I put the other comment. Is that OK?

Reply:
Ah ok, missed that. Thanks!